docs/topics/basic-syntax.md

```diff
@@ -133,44 +133,87 @@ See [Functions](functions.md).
 
 ## Variables
 
-Read-only local variables are defined using the keyword `val`. They can be assigned a value only once.
+In Kotlin, you declare a variable starting with a keyword, `val` or `var`, followed by the name of the variable.
+
+Use the `val` keyword to declare variables that are assigned a value only once. These are read-only local variables that can’t be reassigned a different value
```

Comment:
Consider adding the word immutable as we use mutable for var:
Use the `val` keyword to declare immutable variables....
or something similar to what you wrote at var: 
These are immutable, read-only local variables